### ISSUE TITLE
The project is abandoned?

### ISSUE THREAD

**Issue by Naytaco** [picotorrent/picotorrent#1293]
Question for the developer.
Has the project been abandoned? Or is it still being developed? Will there be updates?

**Comment by SkyleKayma** [comment]
Even without updates, PicoTorrent works flawlessly and is incredibly fast.

**Comment by dakujem** [comment]
Yes, it works well for now, but...

**Comment by Kaktusik234** [comment]
True, PicoTorrent works great, but it would be useful to have information on the current status of the project's development.  I have the impression that the project has been abandoned, as the lack of updates for almost 4 years suggests that work has ceased. In addition, the developer of Pico is currently working on another BitTorrent client: 
https://github.com/porla/porla available, unfortunately, only for Linux.
(Apologies for my English I'm using fat DeepL)

**Comment by vktr** [comment]
Hi!

True, I've lacked the energy and motivation to give PicoTorrent the love I think it deserves. I wouldn't call it abandoned since I'm still around and monitoring things, but I would definitely call it in hibernating mode 😄

**Comment by Kaktusik234** [comment]
Many thanks for the information on the future development of Pico. I am extremely grateful for the answer provided. I hope to return to work on Pico one day.

**Comment by ArcticGems** [comment]
> Even without updates, PicoTorrent works flawlessly and is incredibly fast.

Except for the outdated OpenSSL.

I wish atleast a new update with the latest OpenSSL.

**Comment by rmarquis** [comment]
At least, a new release from current code would be great. I've not been able to build it due to dependency issues and gave up. The last binary release is 4 years old now, but there are many improvements (including dark mode) that are out of reach of regular users.

**Comment by xerz-one** [comment]
ARM64 support would very much be welcome too, the x64 build works but is not ideal... guess I can use Transmission/Fragments on WSL for now tho

**Comment by advaniji** [comment]
> it in hibernating mode

it surely needs an update . given how much its loved by people . there are couple of major bugs

**Comment by advaniji** [comment]
> 've lacked the energy and motivation to give PicoTorrent the love I think it deserves

please make an update ( AI assited update will be welcomed too)

**Comment by tuxayo** [comment]
@vktr would you be open to share the maintainership of the project? Of course only if someone volunteers and has a clear enough track record that would fit the project.
It's fine if after a quick look you don't feel conformable with eventual volunteers.
But at least it would be known that the door is open and it's a matter of showing enough maintainership general background + enough experience with the tech stack.
It's not a light decision to trust someone else with a fairly sized user base.

There are contradictory constrains on the long run
- the xz incident showed that it's not a light decision to share maintainership
  - can be mitigated by not doing it too late to still be able to require a quick glace at merge requests, that would have prevented the attacker to be able to put any binary they liked. Though anyone can make a malicious takeover if having long term timeframe to wait for being alone in charge Not sure there is any solution for that for projects without a large dev base. Which is many projects if they don't have a huge userbase.
- as time passes, current versions of libretorrent, openssl and other dependencies will have more and more known vulnerabilities

**Comment by vktr** [comment]
Hello!

Of course 😄 For anyone interested in maintaining PicoTorrent, I am always happy to review pull requests and discuss roadmaps, etc! The main reason I can't really maintain PicoTorrent currently is that I haven't had a Windows machine for a good few years, but if anyone wants to dig in, feel free!

**Comment by tribals** [comment]
> I am always happy to review pull requests and discuss roadmaps, etc!

Where this discussion should happen?

There is almost 200 issues here to the day. It would be good to triage them and compose some roadmap.

**Comment by rmarquis** [comment]
With all due respect, I believe it should be clear by now that picotorrent is completely unmaitained.

What is needed is someone to step on as maintainer. That's easier said that done, especially that the current source does not seem to compile on a modern windows version due to breaking dependencies (I tried and miserably failed, as I'm not well versed in windows development).

I'm pretty sure vktr would be happy to let someone take the lead, if that someone happens to be motivated enough to do it.

**Comment by vktr** [comment]
Yeah, I'd consider PicoTorrent unmaintained. I haven't had the energy to bring it up to date, and I have no Windows machine to do it on either.

If someone wants to take lead, I am happy to advise or discuss how that can be done, but I wouldn't count on me doing it (taking the lead, that is). Sorry.
